extensions/cl_extension_template.asciidoc

```diff
@@ -387,6 +387,29 @@ extensions, or the optional interactions are documented as Modifications
 to the other extension specs, then this section may be omitted.
 ****
 
+== Conformance tests
+
+****
+This section contains a description of the tests that an implementation
+must pass to be considered conformant with the specification. This section must
+be present for `khr` and `ext` extensions and is recommended for vendor
+extensions.
+
+If it is not possible to provide any tests for the specification, a
+justification must be provided here.
+
+Test descriptions must be numbered and their order maintained across
+successive revisions of the specification.
+
+****
+
+. The new *clShinyNewFunction* entrypoint must be called and succeed.
+. The value returned for +CL_FANCY_QUERY+ must be in the valid range.
```

Comment:
Minor: We should be recommending back-ticks instead of plus signs for mono-space formatting since plus signs only work for asciidoc but not for asciidoctor, but back-ticks work for both.

Reply:
Will do.